api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -638,7 +638,7 @@ private void updatePassword(User user, String newPassword) {
 
 	@Override
 	public void changePassword(User user, String newPassword) throws APIException {
-		dao.changePassword(user, newPassword);
+		updatePassword(user, newPassword);
```

Comment:
The changePassword method that takes old and new password delegates to `changePassword(String user, String oldPassword, String newPassword)` and this already uses the utility method.